src/test/resources/application-integration-test.properties

```diff
@@ -3,4 +3,4 @@ it.wsmUrls.dev=https://workspace.dsde-dev.broadinstitute.org/
 it.wsmEndpoints.workspaces=api/workspaces/v1
 
 it.serviceAccountEmail=william.thunderlord@test.firecloud.org
-it.serviceAccountFilePath=src/test/resources/rendered/wsm-firecloud-account.json
+it.serviceAccountFilePath=src/main/resources/generated/wsm-firecloud-account.json
```

Comment:
Should this one be reading from the test dir since you're rendering to both locations?